cartographer/cloud/internal/handlers/add_rangefinder_data_handler.h

```diff
@@ -32,9 +33,9 @@ DEFINE_HANDLER_SIGNATURE(
     "/cartographer.cloud.proto.MapBuilderService/AddRangefinderData")
 
 class AddRangefinderDataHandler
-    : public async_grpc::RpcHandler<AddRangefinderDataSignature> {
+    : public AddSensorDataHandler<AddRangefinderDataSignature> {
  public:
-  void OnRequest(const proto::AddRangefinderDataRequest &request) override;
+  void OnSensorData(const proto::AddRangefinderDataRequest& request) override;
   void OnReadsDone() override;
```

Comment:
if this is the same for all sensordatahandlers, you could move it to the base class

Reply:
good call. done.